src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/Helper.java

```diff
@@ -349,6 +350,24 @@ public static boolean isClaimantNonSystemUser(CaseData caseData) {
         return true;
     }
 
+    /**
+     * Checks if the respondent is a non-system user.
+     * A non-system user respondent refers to the cases that have been
+     * transferred from legacy ECM or a paper based claim
+     * which a caseworker would manually create in ExUI.
+     *
+     * @param caseData The case data to check for respondent user type.
+     * @return true if the respondent is a non-system user, false otherwise.
+     */
+    public static boolean isRespondentSystemUser(CaseData caseData) {
+        if (caseData != null) {
```

Comment:
This sets default to true. I think default should be false to give privilege only when proven to have system user status. The code now gives that access if CaseData is null.

Reply:
I'm following the same concept as isClaimantNonSystemUser function in the file.
Also this validation is used in Controller in below manner inside ClaimantTellSomethingElseController

        if (Boolean.FALSE.equals(Helper.isRespondentSystemUser(caseData))) {
            caseData.setClaimantTseRespNotAvailable(YES);
        }
        
       so in default it is actually blocking the user by returning true